service/src/main/java/org/apache/fineract/cn/notification/service/internal/config/NotificationConfiguration.java

```diff
@@ -103,23 +108,24 @@ public void configurePathMatch(final PathMatchConfigurer configurer) {
 	}
 	
 	@Bean
-	public PooledConnectionFactory jmsFactory() {
+	public PooledConnectionFactory pooledConnectionFactory() {
 		PooledConnectionFactory pooledConnectionFactory = new PooledConnectionFactory();
 		ActiveMQConnectionFactory activeMQConnectionFactory = new ActiveMQConnectionFactory();
-		activeMQConnectionFactory.setBrokerURL(this.environment.getProperty("activemq.brokerUrl", "vm://localhost?broker.persistent=falseac"));
+		activeMQConnectionFactory.setBrokerURL(this.environment.getProperty("activemq.brokerUrl","vm://localhost?broker.persistent=false"));
 		pooledConnectionFactory.setConnectionFactory(activeMQConnectionFactory);
 		return pooledConnectionFactory;
 	}
 	
 	@Bean
 	public JmsListenerContainerFactory jmsListenerContainerFactory(PooledConnectionFactory jmsFactory) {
-		DefaultJmsListenerContainerFactory factory = new DefaultJmsListenerContainerFactory();
+		final DefaultJmsListenerContainerFactory factory = new DefaultJmsListenerContainerFactory();
+		factory.setConnectionFactory(jmsFactory);
 		factory.setPubSubDomain(true);
 		factory.setConnectionFactory(jmsFactory);
 		factory.setErrorHandler(ex -> {
-			loggerBean().error(ex.getCause().toString());
+			loggerBean().warn(ex.getCause().toString());
 		});
-		factory.setConcurrency(this.environment.getProperty("activemq.concurrency", "1-1"));
+		factory.setConcurrency(this.environment.getProperty("activemq.concurrency","1-1"));
```

Comment:
Are u doing this twice as I see another in ur properties file?

Reply:
Yes, this is a mistake on my part and redundant. I made this error whiles troubleshooting why I couldn't assert the POST_TEMPLATE event. But this has been resolved now

But I want to overwrite the concurrency and max-concurrency to be 1-1. I believe this is how it should be done? Please correct me if I'm wrong